### PR TITLE
[FIX] crm: update the helper store

### PR DESCRIPTION
**Pull request by adsh-odoo**
As the "Create" button is renamed to "New" so update it and remove "top left" in 
helper to enhance clarity and avoid any confusion.          

Task-3420529

### CODE REVIEW

File: addons/crm/i18n/crm.pot [hunk 1]
```diff
@@ -3674,22 +3674,22 @@ msgstr ""
...
 msgstr ""
 
 #. module: crm
 #. odoo-python
 #: code:addons/crm/models/crm_lead.py:0
 #, python-format
 msgid ""
-"Use the top left <i>Create</i> button, or send an email to %s to test the "
+"Use the top left <i>New</i> button, or send an email to %s to test the "
 "email gateway."
```
Comment (niyasraphy) [inline addons/crm/i18n/crm.pot:3686]: and now we have the button in Right side :wink:

File: addons/crm/i18n/crm.pot [hunk 2]
```diff
@@ -3674,22 +3674,22 @@ msgstr ""
 
 #. module: crm
 #: model_terms:ir.ui.view,arch_db:crm.crm_action_helper
-msgid "Use the top left <i>Create</i> button, or send an email to"
+msgid "Use the <i>New</i> button, or send an email to"
```
Comment (tde-banana-odoo) [inline addons/crm/i18n/crm.pot:3677]: Normally in stable we keep the old translation and add the new one (to be cleaned when reaching master and generating final pot files before release). Indeed people could fetch transifex translations without updating the code and they would still need the old translations. Could you add the new one while keeping the old one please ?

Comment (adsh-odoo) [reply addons/crm/i18n/crm.pot:3698]: Hello @tde-banana-odoo 

Suggested changes have been done.

Thanks!!

Comment (tde-banana-odoo) [reply addons/crm/i18n/crm.pot:3698]: Did you generate the file using the tool ? Seems the ordering is not correct, not sure ...

Comment (adsh-odoo) [reply addons/crm/i18n/crm.pot]: Hello @tde-banana-odoo 

You can have a look now.

Thanks!!

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**